src/key_info_managers/mod.rs

```diff
@@ -299,19 +346,19 @@ impl KeyInfoManagerClient {
     /// # Errors
```

Comment:
Line 344 should read `Returns a Vec<ApplicationIdentity> of clients that have keys in this provider`